cmd/minikube/cmd/mount.go

```diff
@@ -225,3 +251,45 @@ func getPort() (int, error) {
 	defer l.Close()
 	return l.Addr().(*net.TCPAddr).Port, nil
 }
+
+func enableCifsShare(hostShareName string, vmDestinationPath string, runner command.Runner) (error) {
+	// Ensure that the current user is administrator because creating a SMB Share requires Administrator privileges.
+	_ , err := pkghyperv.IsWindowsAdministrator()
+	if err != nil {
+		return err
+	}
+
+	hostname, _ := os.Hostname()
+	user, err := pkghyperv.GetCurrentWindowsUser()
+	if err != nil {
+		return err
+	}
+	user = strings.Replace(user,(hostname + "\\"), "",1)
+
+	// Check if the Credential exists in the credential store.
+	var credentialName = "minikube"
+	var password = ""
+	cred, err := wincred.GetGenericCredential(credentialName)
```

Comment:
Any calls to `wincred` should be constrained to files with a `// +build windows` rule.